onnx/checker.cc

```diff
@@ -164,7 +169,8 @@ void check_tensor(const TensorProto& tensor, const CheckerContext& /*ctx*/) {
         check_field(string_data);
         break;
 
-      default:
+    }
+    if (!isChecked) {
```

Comment:
use the "default" to check all other unsupported cases should be good. no?

Reply:
No. As I explained in the commit/PR message `-Wall` includes `-Wswitch` which warns about unhandled enum switch-cases. The problem I fixed here would have not been there (because detected by CI) if `-Wall -Werror` was enabled by CI and `default:` was not used.

It is better to explicitly check all other unsupported cases with e.g.:

```
case TensorProto.Foo:
case TensorProto.Bar:
  break; // Unhandled
```

When now another member is added to the enum, the compiler will warn (and on CI error) that it is not handled. Isn't this a great benefit?